phase_settable.rb

```diff
@@ -0,0 +1,19 @@
+module PhaseSettable
```

Comment:
If you include this module in both the teacher classes and student, then that is a problem. A student should not have access to the ```set_phase_teacher``` method. 

How could this be refactored? Have you looked into how to [use certain methods from a module](http://stackoverflow.com/questions/322470/can-i-invoke-an-instance-method-on-a-ruby-module-without-including-it) and not all methods? Or, what is a different design approach to solve this problem?